vike/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.ts

```diff
@@ -1111,20 +1116,48 @@ async function findPlusFiles(
     assertPosixPath(dir)
     ignorePatterns.push(`${path.posix.relative(userRootDir, dir)}/**`)
   }
-  const result = await glob(`**/+*.${scriptFileExtensions}`, {
-    ignore: [
-      '**/node_modules/**',
-      // Allow:
-      // ```
-      // +Page.js
-      // +Page.telefunc.js
-      // ```
-      '**/*.telefunc.*',
-      ...ignorePatterns
-    ],
-    cwd: userRootDir,
-    dot: false
-  })
+
+  let result: string[] = []
+
+  // If there is no .git folder, this will error
+  try {
```

Comment:
This looks great, do you think we could extract this logic to a separate `glob()` function that we call here?